lib/matplotlib/figure.py

```diff
@@ -1261,38 +1261,57 @@ def draw_artist(self, a):
     def get_axes(self):
         return self.axes
 
-    def legend(self, handles, labels, *args, **kwargs):
+    def legend(self, *args, **kwargs):
```

Comment:
While you are at it, do you mind migrating this to a numpydoc compatible docstring?
(I can also do it if you don't mind me pushing to your branch).